base/freestanding/femr/docker-compose.yaml

```diff
@@ -175,6 +175,13 @@ services:
     networks:
       - internal
 
+  timewarp:
+    image: ghcr.io/uwcirg/fhir-mock-data:${TIMEWARP_IMAGE_TAG:-latest}
+    # include `--profile manual` in docker-compose calls; hiden by default
+    profiles:
+      - manual
```

Comment:
Does timewarp need to be added to cosri-environments to function? timewarp seems like a tool we would run only occasionally, for some deploys

`profiles` is a great way to conditionally include this service, but I'd rather not make the cosri-environments YAML any more complicated if possible

Reply:
i'm open to another process.  how else might we import this Docker container where it will have access to `http://fhir-internal:8080/fhir` ?

Reply:
Just to close the loop, we talked about using the `docker run --network` option to specify the FHIR server `internal` network